slack-base/src/main/java/com/hubspot/slack/client/models/response/chat/ChatDeleteScheduledMessageResponseIF.java

```diff
@@ -0,0 +1,14 @@
+package com.hubspot.slack.client.models.response.chat;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.response.SlackResponse;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
```

Comment:
Do we need this?

Reply:
Right there is no need for that, because there is no response metadata in this endpoint's response. I'll update and fix it.